app/controllers/publishers/vacancies/build_controller.rb

```diff
@@ -2,8 +2,8 @@ class Publishers::Vacancies::BuildController < Publishers::Vacancies::BaseContro
   include Wicked::Wizard
   include OrganisationsHelper
 
-  steps :job_role, :job_role_details, :job_location, :schools, :job_details, :working_patterns, :pay_package,
-        :important_dates, :documents, :applying_for_the_job, :job_summary
+  steps :job_role, :job_role_details, :job_location, :schools, :education_phases, :job_details, :working_patterns,
+        :pay_package, :important_dates, :documents, :applying_for_the_job, :job_summary
```

Comment:
Cesi says

> We fetched your branch and were testing the process of creating a vacancy and solved a bug for which it was asking the education phase when it was not supposed to. you have to add:
```ruby
when :education_phases
  skip_step if vacancy.one_phase?
```
on line 20 in app/controllers/publishers/vacancies/build_controller.rb

Reply:
Maybe this `show` action should use vacancy_step_process instead of rolling its own logic?